src/exercises.js

```diff
@@ -1,15 +1,25 @@
-function filterNumbers(arr, largerThan) {
+/*function filterNumbers(arr, largerThan) {
   return arr.filter(function (number) {
     return number > largerThan;
   });
-}
+}*/
+
+const filterNumbers = (arr, largerThan) => {
+  return arr.filter((number) => number > largerThan);
+};
+
 
-function returnFirstTwoDrivers() {
+/*function returnFirstTwoDrivers() {
   const drivers = ["Sally", "Bob", "Freddy", "Claudia"];
   return drivers.slice(0, 2);
-}
+}*/
+const returnFirstTwoDrivers = () => {
+  const drivers = ["Sally", "Bob", "Freddy", "Claudia"];
+  return drivers.slice(0, 2);
+};
 
-const person = {
+
+/*const person = {
```

Comment:
`const person = {
  name: "Bob",
  greet: function () {
    setTimeout(() => {
      console.log("Hello, my name is " + this.name);
      console.log("👆En el console log tiene que decir: Hello, my name is Bob");
      console.log(this);
    }, 2000);
  },
};
`